rpi5.yaml

```diff
@@ -312,6 +312,7 @@ images:
         gpt_type: B921B045-1DF0-41C3-AF44-4C6F280D3FAE # Linux aarch64 root
         type: raw_image
         size: 8 GiB
+        resize: false
```

Comment:
does it mean that we will get domd image size = 1GB?
If that's true then line size: 8 GB puzzles me. 
I think this is more like a hack so it worth mentioning in the comments. 
I think the best way is to fix moulin or use wic images provided by Yocto.
One more solution I see is to change size setting to be as a variable which is passed to the domd recipe so it can create image with correct size

Reply:
Yes. We will get an 8 GiB partition with a file system of 1 GiB inside and about 7 GiB of inaccessible space.
This is exactly the same behavior we had in this case before `moulin 0.27` when resizing of the ext4 was added.

In fact, `size: 8 GiB` means that rouge will create a partition of the specified size.
In the case of `type: raw_image,` the ext4 image will be put inside that partition as is (before `moulin 0.27`). After version 0.27, moulin tries to resize the ext4 image to fill the entire size of the partition.

Yes, I agree. This commit is a workaround, and the proper fix is to modify our recipe to generate 8 GiB domed.ext4.
I see some solution, so will try it.

For now will put this PR into [WIP] state.